thorlcr/graph/thgraphslave.cpp

```diff
@@ -1280,8 +1280,26 @@ bool CSlaveGraph::serializeStats(MemoryBuffer &mb)
     jobS->querySharedAllocator()->queryRowManager()->reportSummaryStatistics(stats);
 
     IGraphTempHandler *tempHandler = owner ? queryTempHandler(false) : queryJob().queryTempHandler();
+    offset_t sizeGraphSpill = tempHandler ? tempHandler->getActiveUsageSize() : 0;
     if (tempHandler)
-        stats.mergeStatistic(StSizeGraphSpill, tempHandler->getActiveUsageSize());
+        stats.mergeStatistic(StSizeGraphSpill, sizeGraphSpill);
+
+    // calculate peak spill size
+    if (started&&initialized)
+    {
+        unsigned __int64 activeSpillSize = 0;
```

Comment:
related to prev. comment, let's rename to activeTempSize.

OR, could do in next PR, because confusingly queryTempHandler is actually solely for spills, so it would be best to rename to querySpillHandler for clarify.